lib/core/services/eth_wallet_managment_service.dart

```diff
@@ -64,60 +65,52 @@ class EthWalletManagmentService {
 
   Future<UnconfirmedTransaction> sendTransaction(
       String from, String to, amount) async {
+    if (from.substring(0, 2) != "0x") {
+      from = "0x$from";
+    }
+    if (to.substring(0, 2) != "0x") {
+      to = "0x$to";
+    }
     _logger.i("transaction | to=$to");
-    Uri url = Uri.parse("$baseUrl/txs/new?token=$token");
-    final data = {
-      "inputs": [
-        {
-          "addresses": [from]
-        }
-      ],
-      "outputs": [
-        {
-          "addresses": [to],
-          "value": amount
-        }
-      ]
+    Uri url = Uri.parse("$baseUrlWeb3/tx");
+    Map<String, dynamic> tx = {
+      "from": from,
+      "to": to,
+      "amount": amount,
     };
-    var m = await http.post(url, body: jsonEncode(data));
+    var m = await http.post(url, body: jsonEncode(tx), headers: header);
+
+    print(m.body);
+
     UnconfirmedTransaction trs =
-        UnconfirmedTransaction.fromJson(jsonDecode(m.body));
+        UnconfirmedERC20Transaction.fromJson(jsonDecode(m.body));
     return trs;
   }
 
   Future<EthRealTimeWallet> getWalletInfo(String address) async {
+    if (address.substring(0, 2) != "0x") {
+      address = "0x$address";
+    }
     _logger.i("getETHWalletInfo | args: address=$address");
```

Comment:
logging shoueld be at the top

Reply:
I made it beneath the if statement to make sure that I got the address right